resources/views/widgets/graylog.blade.php

```diff
@@ -31,7 +31,8 @@
             return {
                 stream: "{{ $stream }}",
                 device: "{{ $device }}",
-                range: "{{ $range }}"
+                range: "{{ $range }}",
+                loglevel: "{{ $loglevel }}",
```

Comment:
Javascript json does not allow commas on the last item.

Reply:
You are right, changed